### PR TITLE
Making page load timeout configurable.

### PR DESCRIPTION
**Pull request by zebrik**

### CODE REVIEW

File: common/src/main/java/org/jboss/hal/testsuite/finder/FinderNavigation.java [hunk 1]
```diff
@@ -106,6 +109,17 @@ public FinderNavigation(final WebDriver browser, final Class<? extends BasePage>
     }
 
     /**
+     * Sets the timeout in seconds for the application to be loaded in the beginning of the navigation after refresh.
+     * Use carefully in exceptional situations where default timeout of 30 seconds is not sufficient,
+     * e.g. for large domain performance tests.
+     * @param pageLoadTimeout
```
Comment (rhatlapa) [inline common/src/main/java/org/jboss/hal/testsuite/finder/FinderNavigation.java]: Does it have any benefit to have here the `@param pageLoadTimeout` without description? => Give there short description that it represents timeout in seconds even though you can guess it from general description.

File: common/src/main/java/org/jboss/hal/testsuite/util/Console.java [hunk 2]
```diff
@@ -34,15 +34,30 @@
...
         console.browser = browser;
 
         return console;
     }
 
+    /**
+     * Sets the timeout in seconds for the application to be loaded.
+     * Use carefully in exceptional situations where default timeout of 30 seconds is not sufficient,
+     * e.g. for large domain performance tests.
+     * @param pageLoadTimeout
```
Comment (rhatlapa) [inline common/src/main/java/org/jboss/hal/testsuite/util/Console.java]: The same as above.

**Comment by rhatlapa** [comment]
LGTM, just minor improvement in javadoc requested. please update, and squash.

**Comment by zebrik** [comment]
Parameters described in javadoc and change amended.

**Pull request merged**